css/src/components/form/radios.scss

```diff
@@ -0,0 +1,16 @@
+.radios {
+	.radio {
+		margin-top: $radios-spacing;
+	}
+
+	&.radios-vertical {
```

Comment:
It's used to be an example on a doc page for that, isn't it? Why was it removed?

Reply:
Oh, just realized you used it on a docs page. However, I don't think it was a good example, because the text in the examples is way to long so they would stack anyway. What about shortening it to `Lorem ipsum dolor sit amet` so the difference is visible? Or apply the same `Yes/No/Maybe` as in the example above.

<img width="343" alt="image" src="https://user-images.githubusercontent.com/57374379/158293025-aab95d5b-65e6-4e60-a2a8-75d781853d46.png">

Reply:
My original intention was to include a multi-line example on the page, but since the number of examples has been reduced to only two, it may no longer be practical. Updated to a shorter example.